packages/core/src/textures/BaseTexture.js

```diff
@@ -141,12 +141,13 @@ export class BaseTexture extends EventEmitter
          * @member {PIXI.PMA_MODES}
          * @default PIXI.PMA_MODES.UNPACK
          */
-        this.premultiplyAlpha = premultiplyAlpha !== undefined ? premultiplyAlpha : PMA_MODES.UNPACK;
+        this.pmaMode = pmaMode !== undefined ? pmaMode : PMA_MODES.UNPACK;
 
-        if (typeof premultiplyAlpha === 'boolean')
+        if (options.premultiplyAlpha !== undefined)
         {
-            deprecation('v5.2.0', 'premultiplyAlpha is not a boolean anymore, see PIXI.PMA_MODES');
-            this.premultiplyAlpha = Number(premultiplyAlpha);
+            deprecation('5.2.0', 'PIXI.BaseTexture.premultiplyAlpha property has been changed to `pmaMode`'
```

Comment:
I don’t think you need this double deprecation. It’s already in the bundle, just set this.premultiplyAlpha and be done.

Reply:
done